mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/data/RepaymentScheduleRelatedLoanData.java

```diff
@@ -62,6 +62,6 @@ public BigDecimal getTotalFeeChargesAtDisbursement() {
     }
 
     public DisbursementData disbursementData() {
-        return new DisbursementData(null, this.expectedDisbursementDate, this.actualDisbursementDate, this.principal);
+        return new DisbursementData(null, this.expectedDisbursementDate, this.actualDisbursementDate, this.principal, null, null);
```

Comment:
Instead of passing null, null, initialize the chargeid and chargeamount to null and pass those variables which improves readability.